packages/manager/cypress/e2e/core/kubernetes/lke-summary-page.spec.ts

```diff
@@ -11,8 +27,21 @@ const mockKubeconfigResponse = {
 const mockCluster = kubernetesClusterFactory.build();
 const url = `/kubernetes/clusters/${mockCluster.id}`;
 
+const mockNodePools = nodePoolFactory.buildList(2);
+const buildTags = (num: number) => {
+  const tags: string[] = [];
+  for (let i = 0; i < num; i++) {
+    tags.push(randomLabel());
```

Comment:
```suggestion
    tags.push(randomLabel(3));
```

I'm pretty sure that the reason this test is currently flaky (I was also seeing the same error as @dwiley-akamai) is due to the random length of the tag label. If it is a short tag, then we don't render the `...` button. If it's a longer tag, then the tag label will fade out and `...` button will display, even if we only have 1 tag.

To fix the flake in the final test case, we could either a) delete *all* tags or b) set the label length here so it's always a short label.

I tested with `3` and the test passed consistently, while it failed consistently with something like `17`.

![Screenshot 2025-02-05 at 10 43 43 AM](https://github.com/user-attachments/assets/dfc91bfb-42ce-412c-9824-e6098640ac9a)

![Screenshot 2025-02-05 at 10 47 52 AM](https://github.com/user-attachments/assets/928a476a-290f-42be-a2fc-8d539f86893c)